core/chains/evm/config/toml/config.go

```diff
@@ -702,6 +702,72 @@ func (t *HeadTracker) setFrom(f *HeadTracker) {
 	}
 }
 
+type ClientErrors struct {
+	NonceTooLow                       *string `toml:",omitempty"`
+	NonceTooHigh                      *string `toml:",omitempty"`
+	ReplacementTransactionUnderpriced *string `toml:",omitempty"`
+	LimitReached                      *string `toml:",omitempty"`
+	TransactionAlreadyInMempool       *string `toml:",omitempty"`
+	TerminallyUnderpriced             *string `toml:",omitempty"`
+	InsufficientEth                   *string `toml:",omitempty"`
+	TxFeeExceedsCap                   *string `toml:",omitempty"`
+	L2FeeTooLow                       *string `toml:",omitempty"`
+	L2FeeTooHigh                      *string `toml:",omitempty"`
+	L2Full                            *string `toml:",omitempty"`
+	TransactionAlreadyMined           *string `toml:",omitempty"`
+	Fatal                             *string `toml:",omitempty"`
+	ServiceUnavailable                *string `toml:",omitempty"`
+}
+
+func (r *ClientErrors) setFrom(f *ClientErrors) bool {
+	if f == nil {
+		return false
+	}
```

Comment:
This nil check with return arg seems unnecessarily indirect. Can the caller just do the nil check instead?

Reply:
Fixed